src/net.cpp

```diff
@@ -1150,7 +1150,7 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
         }
     }
 
-    bool whitelisted = hListenSocket.whitelisted || IsWhitelistedRange(addr);
+    whitelisted = whitelisted || IsWhitelistedRange(addr);
```

Comment:
I can say for certain that the operator doesn't exist because I tried it first :)

Reply:
Hehe I also remember it doesn't exist - I looked it up a long time ago and IIRC the rationale is that's because `|=` is the same when using only booleans. In any case this is ok as it is.